chef-utils/lib/chef-utils/dsl/platform.rb

```diff
@@ -123,6 +123,17 @@ def centos_platform?(node = __getnode)
       # chef-sugar backcompat method
       alias_method :centos?, :centos_platform?
 
+      # Determine if the current node is CentOS Stream.
+      #
+      # @param [Chef::Node] node the node to check
+      # @since 17.0
+      #
+      # @return [Boolean]
+      #
+      def centos_stream_platform?(node = __getnode)
+        node.dig("lsb", "id") == "CentOSStream"
```

Comment:
I believe this is populated by the `lsb` ohai plugin, which relies on `/usr/bin/lsb_release`, which is provided by `redhat-lsb-core`, which isn't part of the base system and may not necessarily be installed.

Reply:
Yes I realize that. It was discussed that I could write a ruby replacement in ohai for `/usr/bin/lsb_release` so that it wasn't required to have that installed. If you prefer, I can work on that as well so you don't have to rely on it.

Reply:
Yeah that sounds great